tests/outcomes/plot/test_matplotlib/test_revert/main.py

```diff
@@ -0,0 +1,50 @@
+def plot():
+    try:
+        import matplotlib.pyplot as plt
+    except ModuleNotFoundError:
+        return
+
+    fig, ax = plt.subplots(figsize=(1, 2))
+
+    plt.hist([1, 2])
+    ax.hist([1, 5])
+
+    plt.plot([1, 2], [5, 8])
```

Comment:
plot(kind='pie', ...) draws pie chart actually. Drawinf data Should be of type "bar" in this case

Reply:
Without `kind` parameter it is like data type `line`